churchage.py

```diff
@@ -1,32 +1,62 @@
 import streamlit as st 
 import pandas as pd
 
-csvlink = pd.read_csv("ages.csv")
+ADMIN_PASSWORD = "testpassword"
+CSV_FILE_PATH = "ages.csv"
```

Comment:
better to store these in constant variables. Even better to store a hashed version of the admin password in a separate file but that might be overcomplicating things